README.md

```diff
@@ -1,34 +1,18 @@
 # Overview
 
-Describe the intended usage of this charm and anything unique about how this
-charm relates to others here.
-
-This README will be displayed in the Charm Store, it should be either Markdown
-or RST. Ideal READMEs include instructions on how to use the charm, expected
-usage, and charm features that your audience might be interested in. For an
-example of a well written README check out Hadoop:
-http://jujucharms.com/charms/precise/hadoop
-
-Use this as a Markdown reference if you need help with the formatting of this
-README: http://askubuntu.com/editing-help
-
-This charm provides [service][]. Add a description here of what the service
-itself actually does.
-
-Also remember to check the [icon guidelines][] so that your charm looks good
-in the Juju GUI.
+This charm provides [Dex][https://github.com/coreos/dex]. Add a description
+here of what the service itself actually does.
 
 # Usage
 
 Step by step instructions on using the charm:
 
-juju deploy servicename
-
-and so on. If you're providing a web service or something that the end user
-needs to go to, tell them here, especially if you're deploying a service that
-might listen to a non-default port.
-
-You can then browse to http://ip-address to configure the service.
+```
+juju deploy cs:~containers/easyrsa
+juju deploy cs:~containers/dex
+juju add-relation dex easyrsa
```

Comment:
This bit isn't finished either but I was kind of guiding my thoughts this way that we'll need to securely wrap this service behind TLS to ensure it's not susceptible to MITM.